### ISSUE TITLE
Discussion: approval workflow for third-party content on amp.dev

### ISSUE THREAD

**Issue by matthiasrohmer** [ampproject/wg-outreach#15]
From time to time third parties ask to be added as a resource to amp.dev. To give an specific example see ampproject/docs#2331 where mailgun asks to be listed as a tool on amp.dev's tools page. Another section of the site showing third-party tools is the Platform & Vendors page.

So far I've always forwarded such requests to @pbakaus for explicit approval as we (as far as I know) have never set up an explicit workflow how to approve such additions to the site.

I think we should discuss an "official" way how someone is able to get their product shown on amp.dev and document it clearly. The simplest way I can think of is instead of always flagging Paul anyone moderating the docs PRs could flag @ampproject/wg-outreach for review.

**Comment by pbakaus** [comment]
Yep, good point. For now, I've created an issue template on the docs repo for additions that walks through a bunch of questions, and for the approval process I figured the easiest right now would be to loop in the product manager working on the relevant section of AMP to either approve or disapprove.

That's obviously a pretty "Google-heavy" process, so the questions arises how we'll handle if there are PMs outside of Google. One option would be to route to the relevant working group, if there is one (I believe wg-outreach isn't necessarily the right one for non-developer-tools).

/cc @mrjoro @rudygalfi @jasti @nainar
